### PR TITLE
Ucs plugin

### PR DESCRIPTION
**Pull request by muzixing**
update:

ucs_plugin.py: 

- add log option info

README: 

- fixed error info
- add dependencies info
- add more detailed info

### CODE REVIEW

**Comment by muzixing** [comment]
Hi @marcsolanas,  please review the code.

**Comment by muzixing** [comment]
New modifications:

- use ucs as tag name
- format data as dictionary.

**Comment by muzixing** [comment]
@marcsolanas , the code has been updated, please review it.

File: ucs/README.md [hunk 1]
```diff
@@ -16,37 +28,58 @@ UCS plugin supports to collect data from Cisco Unified Computing Server.
...
-      -z [ZEUS], --zeus [ZEUS]
-                            IP or host name of ZEUS server. (default: 127.0.0.1)
+        UCS = '127.0.0.1'
+        UCS_USER = 'ucspe'
+        UCS_PASSWD = 'ucspe'
+        IS_SECURE = True
+        PORT = 443
+        LOG_LEVEL = 'INFO'
+        
+        ZEUS_TOKEN = "abcde437t80wepepwvees1oylrpcef80"
```
Comment (marcsolanas) [inline ucs/README.md]: Lets remove the bucket token from here

Comment (marcsolanas) [inline ucs/README.md]: Replace with:
<INSERT_TOKEN_HERE>

File: ucs/config.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+UCS_PASSWD = 'ucspe'
+
+# For HTTPS connection, please set IS_SECURE=True, and set PORT, default port number is 443
+# For HTTP connection, pleases et IS_SECURE=False and set PORT, default port number is 80.
+IS_SECURE = True
+PORT = 443
+
+LOG_LEVEL = 'INFO'
+
+ZEUS_TOKEN = "837t80wepepwvees1oylrpcef80ceteg"
```
Comment (marcsolanas) [inline ucs/config.py]: ditto

File: ucs/test/config/config.py [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 UCS_PASSWD = 'ucspe'
+
+# For HTTPS connection, please set IS_SECURE=True, and set PORT, default port number is 443
+# For HTTP connection, pleases et IS_SECURE=False and set PORT, default port number is 80.
 IS_SECURE = False
 PORT = 80
+
 LOG_LEVEL = 'ERROR'
 
 ZEUS_TOKEN = "837t80wepepwvees1oylrpcef80ceteg"
```
Comment (marcsolanas) [inline ucs/test/config/config.py]: ditto

File: ucs/README.md (hunk 1 above)
Comment (muzixing) [inline ucs/README.md]: OK

File: ucs/config.py (hunk 2 above)
Comment (muzixing) [inline ucs/config.py]: oK

File: ucs/test/config/config.py (hunk 3 above)
Comment (muzixing) [inline ucs/test/config/config.py]: OK

**Pull request merged**